Userland/Services/SpiceAgent/VDAgent.h

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright (c) 2021, Kyle Pereira <kyle@xylepereira.me>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include "ClipboardServerConnection.h"
+#include <AK/ByteBuffer.h>
+#include <AK/Vector.h>
+#include <LibCore/Notifier.h>
+
+#pragma once
+
+// Fun fact, I have no idea what VD stands for.
+class VDAgent {
```

Comment:
Can we be consistent then? the service is named SpiceAgent, and you use Spice as the prefix for all the other main objects, so how about just calling this `Agent` or `SpiceAgent`?